src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+This design document will try to ensure that a round with a Rev antag follows this pattern.
+
+### Revolutionaries Gamemode Goals
+
+The goals that we want to reach with the Revolutionaries are the following:
+- Revolutionaries should be applicable as its own round type and as a mid-round antag, equivalent to Nukies/LoneOp or Traitors/Mid-round Traitors. The difference should be in the number of antags and the resources available.
+- Round removal should be discouraged where possible. It should not be the optimal choice to gib or leave a player to rot unless in specific circumstances.
+- Revolutionaries should not be discovered through *no fault of their own* in the Initial Conversions phase (i.e. metagameable mechanics). 
+- A single non-Security crew member discovering an antag should not spell the end of the antag faction's progression in the round.
+- Round end conditions should be clear and straight-forward, though a crew overcoming the antag shouldn't necessarily mean the end of the round.
+- Crew shouldn't be able to hard counterplay the antag without engaging with the antag. 
+
+
+## New Revolutionaries, Basic layout
+<sup>Any values given here are suggestions and are especially marked by being **bolded**. Any such value should be adjusted based on playtesting.</sup>
+
+The Revolutionary antag begins either as a 2-4 roundstart antag roll for the Revolutionary gamemode, or a single crewmember secretly becomes a Revolutionary antag during another gamemode midround.
+
+Revolutionary antags that start out this way are considered Head Revolutionaries (HeadRevs). A Head Revolutionary has the duty to begin the conversion of other crew and have some extra tools at their disposal to strengthen the progression of the antag faction, but they are not strictly necessary for the revolution antags to succeed. The goal for Head Revolutionaries is to convert crew to grow the faction's strength, convert department Heads and survive until the evacuation shuttle without being deconverted. 
+
+Head Revolutionaries share the same team and when they are given the role they obtain the names and jobs of the other Head Revolutionaries on the station. Revolutionaries can be identified by other revolutionaries via an icon next to their character, however Head Revs do not have a unique icon.
+
+For crew, the goal is to deconvert revolutionary converts, protect Heads from being converted, and ensure Head Revs are discovered and/or do not make it onto the evacuation shuttle. 
+
+### Win Conditions
+
+Win conditions for a round with a Revolutionary antag are the following:
```

Comment:
So I guess the logic is that Revs is not about taking the station at all, but... just having RR'd or arrested/converted heads?

Why should Revs bother to convert heads? The difference is just major vs minor victory. 

If Heads know they can deny a Rev major victory, what stops Heads deliberately fleeing the station via a jetpack or a shuttle?

Reply:
> Why should Revs bother to convert heads? 

Not only is a Command rev *very* useful if done quietly, they contribute to the DDD percentages as well. I disagree with the notion that players won't try for the major win as an initial position.

> If Heads know they can deny a Rev major victory, what stops Heads deliberately fleeing the station via a jetpack or a shuttle?

You'd need >50% of Command to do this to deny a Rev major. If Command insists on being shuttle cowards, Revs can still opt for a minor.